### ISSUE TITLE
Run tests for tutorials in examples/*

### ISSUE THREAD

**Issue by benlorenz** [oscar-system/Polymake.jl#268]
Either in this repository or in a separate oscar-system repository (Tests probably via github-actions)

cc: @kalmarek @micjoswig

**Comment by kalmarek** [comment]
yep, I raised this issue when the location of the notebooks was discussed on Thursday meeting; I'm 100% for making sure those work. Do you think `Literate.jl` an option for this?

**Comment by benlorenz** [comment]
`Literate.jl` cannot really do what we need right now, it can create a notebook from a julia file but not the other way round.

But it could be an option if we change the notebooks to `.jl` files, which would bring some advantages:
- `Literate.jl` can convert to jupyter notebooks for publishing them
- `ipynb` is rather bad for version control, `jl` is a lot better
- we can just run the scripts in the github actions
- we could easily convert them to markdown pages instead of relying on github showing the ipynb more or less nicely
disadvantages:
- needs one more step when running them from the source
- cannot be edited from within jupyter

opinions @kalmarek @IvanSpirandelli @saschatimme ?

**Comment by alexej-jordan** [comment]
in my exys, the advantages overweigh the disadvantages. especially now, in context of the coming documentation, the generated markdown files sound promising.

but we probably need to exclude the polydb notebook from the tests.
